autoprotocol/protocol.py

```diff
@@ -5076,6 +5080,9 @@ def provision(self, resource_id, dests, amounts=None, volumes=None):
           one should be specified explicitly in a list matching the order of the
           specified destinations.
           Note:  Volumes and amounts arguments are mutually exclusive. Only one is required
+        informatics: list(dict)
```

Comment:
See above comment. This should be `list(Informatics)`. We could also additionally support `list(dict)` if that's desired.

Reply:
Oh. I think I totally missed/misunderstood this. I thought we wanted to keep user input as a dict. my bad!